modules/ROOT/pages/database-administration/standard-databases/create-databases.adoc

```diff
@@ -104,7 +104,7 @@ The `CREATE DATABASE` command can have a map of options, e.g. `OPTIONS {key: 'va
 | `use`
 |
 Controls how the system handles existing data on disk when creating the database.
-Currently this is only supported with `existingDataSeedInstance` and must be set to `use` which indicates the existing data files should be used for the new database.
+Currently, this is only supported with `existingDataSeedInstance` and must be set to `use` which indicates the existing data files should be used for the new database.
```

Comment:
There is also `seedURI` now isn't there? I'm not sure if that requires `existingData` or not though. I think it probably should.